kibana-reports/public/components/report_definitions/report_settings/__tests__/report_settings_helpers.test.tsx

```diff
@@ -13,6 +13,7 @@
  * permissions and limitations under the License.
  */
 
+import { exception } from 'console';
```

Comment:
I didn't see this is used anywhere, can you remove?

Reply:
Good catch- must have been added when I selected wrong drop-down option in tests, will remove